packages/neuron-ui/src/locales/en.json

```diff
@@ -11,7 +11,9 @@
       "history": "History",
       "addresses": "Addresses",
       "settings": "Settings",
-      "nervos-dao": "Nervos DAO"
+      "nervos-dao": "Nervos DAO",
+      "fail-to-fetch-tip-block-number": "Connection error",
+      "sync-not-start": "Synchronization not started yet"
```

Comment:
Will this fit as one line within the sidebar?

Reply:
> Will this fit as one line within the sidebar?

This message is still wrapped in the sidebar

Reply:
I see. Let's make it shorter.